### PR TITLE
Implement StringValue func for each type.

### PR DESCRIPTION
**Pull request by katsew**

### CODE REVIEW

File: datasource/value.go [hunk 1]
```diff
@@ -1,15 +1,60 @@
...
+
+	// @todo Find more better way to convert...
 	switch cv.ColumnType {
-	// TODO: additional string reprensentation for specific value type
+	case ColumnTypeInt:
+		switch cv.Value.(type) {
+		case int:
+			if i, ok := cv.Value.(int); ok {
+				return strconv.Itoa(i)
+			} else {
```
Comment (castaneai) [inline datasource/value.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

File: datasource/value.go [hunk 2]
```diff
@@ -1,15 +1,60 @@
...
+				return fmt.Sprintf("%d", cv.Value)
+			}
+		case int64:
+			if i, ok := cv.Value.(int64); ok {
+				return strconv.FormatInt(i, 10)
+			} else {
```
Comment (castaneai) [inline datasource/value.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

File: datasource/value.go [hunk 3]
```diff
@@ -1,15 +1,60 @@
...
+				return fmt.Sprintf("%d", cv.Value)
+			}
+		case float64:
+			if f, ok := cv.Value.(float64); ok {
+				return strconv.FormatFloat(f, 'f', -1, 64)
+			} else {
```
Comment (castaneai) [inline datasource/value.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

File: datasource/value.go [hunk 4]
```diff
@@ -1,15 +1,60 @@
...
+				return fmt.Sprintf("%f", cv.Value)
+			}
+		default:
+			return fmt.Sprintf("%d", cv.Value)
+		}
+	case ColumnTypeFloat:
+		if f, ok := cv.Value.(float64); ok {
+			return strconv.FormatFloat(f, 'f', -1, 64)
+		} else {
```
Comment (castaneai) [inline datasource/value.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

**Comment by katsew** [comment]
Clouse due to this fix: https://github.com/Mitu217/tamate/pull/100

**Pull request closed without merging**